scripts/get_digest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+IMAGE=$1
+TAG=$2
+
+CURL=`which curl`
+JQ=`which jq`
+SED=`which sed`
+
+HOST_REGISTRY="https://hub.docker.com/v2"
+PATH="repositories/$IMAGE/tags"
+
+DIGEST=`$CURL -f --silent --header "Accept: application/json" $HOST_REGISTRY/$PATH | $JQ ".results[] | select(.name == \"$TAG\") | .images[].digest" | $SED -e 's/\"//g'`
+
```

Comment:
I would add some testing on the `DIGEST` string. for example, it must have a specific length right?

Reply:
Test added